core/jobs/batch_jobs/mailchimp_population_jobs.py

```diff
@@ -89,8 +92,16 @@ def _get_mailchimp_class() -> mailchimp3.MailChimp:
     # The following is a class initialized in the library with the API key and
     # username and hence cannot be tested directly. The mailchimp functions are
     # tested with a mock class.
+    mailchimp_api_key: Optional[str] = secrets_services.get_secret(
+        'MAILCHIMP_API_KEY')
+    if not mailchimp_api_key:
+        logging.exception('Cloud Secret Manager is not working.')
+        mailchimp_api_key = feconf.MAILCHIMP_API_KEY
```

Comment:
Not sure why we need the fallback. Are we still planning on setting the feconf key in the future?

Reply:
We will remove it after we verify it works.